final_task/rss_reader/rss_reader.py

```diff
@@ -0,0 +1,132 @@
+import argparse
+import feedparser
+import os
+import pdfkit
+import random
+import dominate
+from dominate.tags import *
```

Comment:
It is better to aviod wildcard import whenever you can.

Reply:
Fixed